docs.html

```diff
@@ -725,6 +726,32 @@ <h1 class="display-title">Navbar</h1>
             </iframe>
           </div>
         </section>
+        <section class="display-code-content" id="modal">
+          <h1 class="display-title">Modal</h1>
+          <button class="btn btn-primary btn-modal">Show Modal</button>
+          <div class="modal-container">
+            <h1 class="modal-title">Modal Title</h1>
```

Comment:
There should be only one `h1` per page, read about heading hierarchy

Reply:
Oh Yes will change it to h3 or h4 and then merge is it ok?